app/src/main/res/layout/layout_weather_map.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/cardView_map"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:cardElevation="@dimen/dp_30"
+    android:backgroundTint="@color/cardMapBackground"
```

Comment:
Đặt tên cho các resource cũng cần thống nhất

Reply:
@gianghv-3456 đối với color anh đặt toàn bộ là camel case ấy em